appveyor.yml

```diff
@@ -0,0 +1,16 @@
+---
+clone_depth: 10
+install:
+  - SET PATH=C:\Ruby%ruby_version%\bin;%PATH%
+  - appveyor DownloadFile http://dl.bintray.com/oneclick/OpenKnapsack/x64/openssl-1.0.1l-x64-windows.tar.lzma
```

Comment:
I agreed. I try to build openssl-1.1.0 binary used RubyInstaller recipe as https://github.com/oneclick/knapsack-recipes/tree/master/openssl later.